apps/modernization-ui/src/apps/search/patient/patientSearch/PatientSearch.tsx

```diff
@@ -286,8 +287,14 @@ export const PatientSearch = ({ handleSubmission, personFilter, clearAll }: Pati
                         className="width-full clear-btn"
                         type={'button'}
                         onClick={() => {
-                            form.reset({}, { keepDefaultValues: true });
-                            clearAll();
+                            // Because of the customized nature of Date picker errors, the date picker doesn't reset when the clear all is clicked.
```

Comment:
suggestion:  If the root cause of the issue is due to the customized errors then a ticket should be created to address this permanently.  The majority of our components do not validate themselves so that the caller can choose to enforce specific validations.

Reply:
Yes, agreed and I wanted to see if I could integrate the error in a "rule and validation" way for the controller instead of the custom error way. I can just increase the scope of the story by 1 or 2 points to check that. Will do it.

Reply:
Update on this. Tried to see if changing the datePickerInput's design to show error through "rule and validation" was quick, but it was consuming more time than necessary. This may be something that's not very worth it.

Reply:
I'll get a story created to make sure we can handle this in the future